.circleci/config.yml

```diff
@@ -22,16 +22,16 @@ jobs:
     working_directory: ~/project
     machine:
       enabled: true
-      # TODO: why is this using Ubuntu instead of Debian?
-      image: ubuntu-2004:202010-01
+      # Because we are running Docker inside this job, we need a machine type.
+      # CircleCI does not currently offer Debian machine images.
+      image: ubuntu-2004:2023.07.1
     environment:
       DOCKER_API_VERSION: 1.23
     steps:
       - checkout
       - run:
           name: Install dependencies
           command: |
-            (cd $(pyenv root); git pull)
```

Comment:
Removed because, as written, it [does not work](https://app.circleci.com/pipelines/github/freedomofpress/securedrop-sdk/1351/workflows/cd62afd7-4440-4e55-96ce-4f96662f89dd/jobs/2433/parallel-runs/0/steps/0-102) with the latest machine image. It's not clear to me that we _want_ to update to the latest development version here, in any case, rather than ensuring we're using an up-to-date image. The image we're switching to here does ship a much more recent version of pyenv.